github/apps_installation.go

```diff
@@ -10,10 +10,15 @@ import (
 	"fmt"
 )
 
+type ListRepositories struct {
```

Comment:
Can you please add a godoc-style comment here?  Maybe something like this?

```
// ListRespositories represents the response from the list repos endpoints.
```

Reply:
Added the godoc-style comment!